themes/gauge_theme/static/img/twitter.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please refer this svg from `preview.gauge.org`

Reply:
`preview.gauge.org` is a staging site, if these are to be externalized, they should be via `gauge.org`, but that is not yet updated.

I say let docs hold it's assets, and website can hold it's own. These are small enough to be inlined. Thoughts?